modular_skyrat/modules/bitrunning/code/virtual_domains/ancient_milsim/fluff.dm

```diff
@@ -0,0 +1,81 @@
+/obj/item/tape/ruins/ancient_milsim
+	icon_state = "tape_blue"
+	name = "Developer Commentary"
+	desc = "Tape recording made, presumably, somewhere around twenty third century - at least, that's what the smudged labels are saying. Why is it smudged? Presumably to be authentic."
+
+
+/obj/item/tape/ruins/ancient_milsim/devlog_one
+	name = "Developer Commentary, Log 1: Setting"
+	used_capacity = 160
+
+	storedinfo = list(
+		1 = "<span class='game say'><span class='name'>The universal recorder</span> <span class='message'>says, \"<span class='tape_recorder '>Recording started.</span>\"</span></span>",
+		2 = "<span class='game say'><span class='name'>Unknown person</span> <span class='message'>blares, \"<span class=' '>Greetings, my name is Leon Gardner, and welcome to the beta test of a long awaited sequel to 'Time Of Valor: Border Rescue' - 'Time Of Valor 2'.</span>\"</span></span>",
+		3 = "<span class='game say'><span class='name'>Leon Gardner</span> <span class='message'>states, \"<span class=' '>After three years of development, hopefully it will be worth the wait.</span>\"</span></span>",
+		4 = "<span class='game say'><span class='name'>Leon Gardner</span> <span class='message'>clarifies, \"<span class=' '>In the following pre-release beta test, you'll be completing the intro level of the game, detailing the early stages of the Border War, and introducing you to the new mechanics like Smart NPCs and full molecular simulation.</span>\"</span></span>",
```

Comment:
The way those spans are done kinda bugs me, don't we have a macro that's like `span_game_say()`?

Reply:
I don't really know why, but all the upstream tapes are done that way too, -probably for a reason-, so I've decided not to touch it.